### PR TITLE
Add an example in video processer documentation for loading external images

### PR DESCRIPTION
**Pull request by ldai1**
**Issue #:**
#1525 
**Description of changes:**
Add an example in video processer documentation for loading external images
**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? N/A
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/10_Video_Processor.md [hunk 1]
```diff
@@ -143,6 +143,39 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
...
+      elem.onload = (): void => resolve(elem);
+      elem.onerror = reject;
+      elem.src = url;
+      // to configure CORS access for the fetch of the new image if it is not hosted on the same server
+      elem.crossOrigin = "anonymous";
+    });
+  }
+  async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
+    const canvas = buffers[0].asCanvasElement();
+  // copy the frame to the intermediate canvas
```
Comment (ltrung) [inline guides/10_Video_Processor.md:170]: indent

File: guides/10_Video_Processor.md [hunk 2]
```diff
@@ -143,6 +143,39 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
 }
 ```
 
+More example about building a customized processor to load an external image:
```
Comment (hensmi-amazon) [inline guides/10_Video_Processor.md:146]: Can this be in a different section at the `###` level? i.e.
```
### Example: Building a resize processor
...

### Example: Building a external image overlay processor
...
```

Also this is a bit of a strange use case as is (just replacing the buffer).  I understand from linked issue that its more about CORS, but we should make that more clear.  Or make this an overlay processor which I would prefer.  If a builder really wanted to inject a static image they wouldn't bother with the processor, they would just create a MediaStream and inject directly.

File: guides/10_Video_Processor.md [hunk 3]
```diff
@@ -145,6 +145,41 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
...
+  async function loadImage(url: string, elem: HTMLImageElement): Promise<HTMLImageElement> {
+    return new Promise((resolve, reject) => {
+      elem.onload = (): void => resolve(elem);
+      elem.onerror = reject;
+      elem.src = url;
+      // to configure CORS access for the fetch of the new image if it is not hosted on the same server
+      elem.crossOrigin = "anonymous";
+    });
+  }
+  async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
```
Comment (devalevenkatesh) [inline guides/10_Video_Processor.md:171]: nit: Need a newline.

File: guides/10_Video_Processor.md [hunk 4]
```diff
@@ -145,6 +145,41 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
...
+  private targetCanvas: HTMLCanvasElement = document.createElement('canvas') as HTMLCanvasElement;
+  private targetCanvasCtx: CanvasRenderingContext2D = this.targetCanvas.getContext('2d') as CanvasRenderingContext2D;
+  private canvasVideoFrameBuffer = new CanvasVideoFrameBuffer(this.targetCanvas);
+
+  private renderWidth: number = 0;
+  private renderHeight: number = 0;
+  private sourceWidth: number = 0;
+  private sourceHeight: number = 0;
+  
+  async function loadImage(url: string, elem: HTMLImageElement): Promise<HTMLImageElement> {
```
Comment (devalevenkatesh) [inline guides/10_Video_Processor.md:162]: We have the function here, but should we explain or add a bit on where to add this function as an listener or comment for clarifying?

**Review by devalevenkatesh — APPROVED** [review]

**Review by hensmi-amazon — APPROVED** [review]
nice!

**Pull request merged**